### ISSUE TITLE
Output file 

### ISSUE THREAD

**Issue by nakeene** [qichao1984/NCyc#3]
Hello,

What are the units of the second column in the output file? 
See image attached. 

![image](https://user-images.githubusercontent.com/16257684/52539278-d3bcca00-2d41-11e9-8c6c-c5292e9cd207.png)

**Comment by rprops** [comment]
Yes, those are counts (i.e. number of genes with homology to the gene in the first column that were retrieved in your (meta)genome dataset).
